src/upload/system/library/transbank/utils/TransbankSdkWebpay.php

```diff
@@ -58,17 +55,10 @@ public function commitTransaction($tokenWs)
         $result = [];
         try {
             $this->log->logInfo('getTransactionResult - tokenWs: ' . $tokenWs);
-            if ($tokenWs == null) {
-                throw new \Exception("El token webpay es requerido");
-            }
-
             return (new Transaction)->commit($tokenWs);
-        } catch (\Exception $e) {
-            $result = [
-                "error" => 'Error al confirmar la transacción',
-                "detail" => $e->getMessage()
-            ];
-            $this->log->logError(json_encode($result));
+        } catch (TransactionCommitException $e) {
+            $this->log->logError(json_encode($e->getMessage()));
```

Comment:
same questions here

Reply:
It is not necessary, the corresponding modifications will be made